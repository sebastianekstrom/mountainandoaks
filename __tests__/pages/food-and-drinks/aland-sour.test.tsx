import React from "react";
import { render, screen } from "@testing-library/react";
import LandSour from "pages/food-and-drinks/aland-sour";

describe("AlandSour page", () => {
  it("renders title correctly", () => {
    render(<LandSour />);
    const title = screen.queryAllByText(/Ã…land Sour/i);
    expect(title).toBeDefined();
  });

  it("renders the ingress text", () => {
    render(<LandSour />);
    const ingress = screen.queryAllByText(
      /A refreshing gin cocktail where the crisp flavor of fresh pressed apple takes the lead, rounded out with bright citrus and a touch of elderflower sweetness../i,
    );
    expect(ingress).toBeDefined();
  });

  it("renders the correct ingredients", () => {
    render(<LandSour />);
    const ingredientList = screen.getByTestId("INGREDIENT_LIST");
    expect(ingredientList).toMatchSnapshot();
  });
});
