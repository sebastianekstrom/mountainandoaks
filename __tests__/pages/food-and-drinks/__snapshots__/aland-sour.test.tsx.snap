// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlandSour page renders the correct ingredients 1`] = `
<section
  class="mb-8 lg:mb-0"
  data-testid="INGREDIENT_LIST"
>
  <div>
     
  </div>
  <div
    class="flex flex-col"
  >
    <ul>
      <li
        class="flex justify-between mb-1 transition-opacity "
      >
        <div
          class=""
        >
          <p
            class="variable font-sans text-paragraph text-regular leading-paragraph font-semibold"
          >
            Gin
          </p>
        </div>
        <p
          class="variable font-sans text-paragraph text-regular leading-paragraph "
        >
          5
          cl
        </p>
      </li>
      <li
        class="flex justify-between mb-1 transition-opacity "
      >
        <div
          class=""
        >
          <p
            class="variable font-sans text-paragraph text-regular leading-paragraph font-semibold"
          >
            Lemon juice
          </p>
        </div>
        <p
          class="variable font-sans text-paragraph text-regular leading-paragraph "
        >
          3
          cl
        </p>
      </li>
      <li
        class="flex justify-between mb-1 transition-opacity "
      >
        <div
          class=""
        >
          <p
            class="variable font-sans text-paragraph text-regular leading-paragraph font-semibold"
          >
            Elderflower syrup
          </p>
        </div>
        <p
          class="variable font-sans text-paragraph text-regular leading-paragraph "
        >
          3
          cl
        </p>
      </li>
      <li
        class="flex justify-between mb-1 transition-opacity "
      >
        <div
          class=""
        >
          <p
            class="variable font-sans text-paragraph text-regular leading-paragraph font-semibold"
          >
            Fresh pressed apple juice
          </p>
        </div>
        <p
          class="variable font-sans text-paragraph text-regular leading-paragraph "
        >
          15
          cl
        </p>
      </li>
    </ul>
  </div>
</section>
`;
